pyproject.toml

```diff
@@ -43,6 +43,7 @@ issues = "https://github.com/Colin-b/httpx_auth/issues"
 
 [project.optional-dependencies]
 testing = [
+    "pytest==8.3.4",
```

Comment:
I am not sure why this was missing here, I assume it comes with `pytest-cov` in whatever version is deemed suitable there.